pom.xml

```diff
@@ -180,6 +192,8 @@
                     <excludeVulnerabilityIds>
                         <!-- CWE-400: Uncontrolled Resource Consumption ('Resource Exhaustion') (7.5); from org.apache.spark:spark-sql_${spark.version} -->
                         <exclude>CVE-2023-33546</exclude>
+                        <!-- CWE-300: Channel Accessible by Non-Endpoint ('Man-in-the-Middle') (6.5); from org.apache.zookeeper:zookeeper:jar:3.7.2:compile -->
+                        <exclude>CVE-2023-4586</exclude>
```

Comment:
Add a short justification for excluding the vulnerability

Reply:
No versions of zookeeper which fixes this issue? :)